packages/query/src/graphql/plugins/PgConnectionArgFirstLastBeforeAfter.ts

```diff
@@ -41,11 +41,13 @@ export default (builder) => {
         return {
           pgQuery: (queryBuilder: QueryBuilder) => {
             if (!unsafe) {
-              queryBuilder.limit(MAX_RECORD_COUNT);
+              queryBuilder.limit(MAX_ITEM_COUNT);
```

Comment:
what's the difference between `queryBuilder.limit` and `queryBuilder.first`?

Reply:
> It's not actually silent, user can compare the result size with totalCount. I think not throwing is alright.
Alright i'll change it back so that it clamps instead

Reply:
i shouldn't be using `limit` here I think. I have to check what graphql defaults to for an unspecified first/last